src/styles/blocks/buy-reasons.scss

```diff
@@ -0,0 +1,63 @@
+@import "../utils/vars";
+@import "../utils/mixins";
+
+.buy-reasons {
+  @include set-horizontal-padding;
+
+  &__ {
+    &layout {
+      margin-top: $site-parts-top-margin;
+
+      @include for-tablet {
+        @include set-tablet-grid;
+      }
+
+      @include for-desktop {
+        @include set-desktop-grid;
+      }
+    }
+
+    &title {
+      @include style-title;
+
+      @include for-tablet {
+        width: 287px;
+        grid-column: 1 / 3;
+      }
+
+      @include for-desktop {
+        width: 411px;
+        grid-column: 1 / 5;
+      }
+    }
+
+    &content {
+      @include for-tablet {
+        grid-column: 4 / -1;
+      }
+
+      @include for-desktop {
+        grid-column: 6 / 11;
+      }
+    }
+
+    &why {
+      margin-top: 16px;
+
```

Comment:
redundant white space

Reply:
I deleted new lines between atributes in css classes.